tests/Builder/Accumulator/ConcatArraysAccumulatorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Accumulator;
+
+use MongoDB\Builder\Accumulator;
+use MongoDB\Builder\Expression;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+/**
+ * Test $concatArrays accumulator
+ */
+class ConcatArraysAccumulatorTest extends PipelineTestCase
+{
+    public function testWarehouseCollection(): void
+    {
+        $concatArrays = Accumulator::concatArrays(...);
```

Comment:
Ha, our code from discussing different ways to write agg pipelines found its way in here. I'll let you decide whether you want to keep it this way or inline it.